pr_insight/tools/pr_line_questions.py

```
@@ -7,7 +7,7 @@
 from pr_insight.algo.ai_handlers.base_ai_handler import BaseAiHandler
 from pr_insight.algo.ai_handlers.litellm_ai_handler import LiteLLMAIHandler
 from pr_insight.algo.git_patch_processing import (
-    convert_to_hunks_with_lines_numbers, extract_hunk_lines_from_patch)
+    decouple_and_convert_to_hunks_with_lines_numbers, extract_hunk_lines_from_patch)
```

Ruff (0.8.2)

`pr_insight.algo.git_patch_processing.decouple_and_convert_to_hunks_with_lines_numbers` imported but unused

(F401)